### PR TITLE
feat: meta/anilist fetchStaffById

### PR DESCRIPTION
**Pull request by irfanshadikrishad**
**What kind of change does this PR introduce?**

Implemented fetchStaffById in meta/anilist

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
No

**Summary**

**Other information**

### CODE REVIEW

File: src/models/types.ts [hunk 1]
```diff
@@ -478,3 +478,32 @@ export interface ProxyConfig {
...
+  type?: string;
+  image: {
+    extraLarge?: string;
+    large?: string;
+    medium?: string;
+  };
+  color?: string;
+}
+
+export interface IStaffInfo {
```
Comment (riimuru) [inline src/models/types.ts:494]: can you rename this to `IStaff`

**Review by riimuru — CHANGES_REQUESTED** [review]
Thank you for your pr! Everything else looks great

File: src/models/types.ts (hunk 1 above)
Comment (irfanshadikrishad) [reply src/models/types.ts]: Done brother.

**Review by riimuru — APPROVED** [review]

File: src/models/types.ts (hunk 1 above)
Comment (riimuru) [reply src/models/types.ts]: Thank youu!

**Pull request merged**